src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentEventHandlerLoweringPass.cs

```diff
@@ -222,5 +263,49 @@ private static IReadOnlyList<IntermediateToken> GetAttributeContent(TagHelperDir
                 return node.FindDescendantNodes<IntermediateToken>();
             }
         }
+
+        private IntermediateNode RewriteParameterUsage(IntermediateNode parent, TagHelperDirectiveAttributeParameterIntermediateNode node)
+        {
+            // Now rewrite the node to look like:
+            //
+            // builder.AddEventPreventDefaultAttribute(2, "onclick", true); // If minimized.
```

Comment:
Is the `true` the return value of `onclick`? If so, wouldn't it have to be `false`?

```suggestion
            // builder.AddEventPreventDefaultAttribute(2, "onclick", false); // If minimized.
```

Reply:
It's not the return value of onclick. It indicates whether it should enable preventDefault or not. So a minimized attribute means, the value is true by default.